### ISSUE TITLE
AttributeError: 'module' object has no attribute 'make_response_from_query_sets'

### ISSUE THREAD

**Issue by kinwolfqc** [chfw/Flask-Excel#3]
I always get this error trying to use the documented 'make_response_from_query_sets

I use the exemple in the doc to try to run it

@admin.route('/exportcsv')
def export_csv():
    query_sets = Classes.query.filter_by(id=1).all()
    column_names = ['id', 'name','participants']
    return excel.make_response_from_query_sets(query_sets, column_names, "xls")

**Comment by chfw** [comment]
Right, first of all, thanks for using Flask-Excel. You are using un-released version 0.0.2. 

So, you will need to make sure you have these un-released versions as well: pyexcel-webio v0.0.2 , pyexcel v0.1.5 and pyexcel-xls 0.0.7. Here's what you need to do:

```
  - pip install git+https://github.com/chfw/pyexcel-io.git  
  - pip install git+https://github.com/chfw/pyexcel.git
  - pip install git+https://github.com/chfw/pyexcel-webio.git
  - pip install git+https://github.com/chfw/pyexcel-xls.git
```

Those packages will be released once I will have finished relevant documentation and unit tests.

**Comment by kinwolfqc** [comment]
Thanks, I never noticed the version were wrong on those package
